api/v1beta1/aerospikecluster_validating_webhook.go

```diff
@@ -254,6 +255,42 @@ func (c *AerospikeCluster) validate(aslog logr.Logger) error {
 		return err
 	}
 
+	if err := c.validateSCNamespaces(); err != nil {
```

Comment:
validateSCNamespaces is across racks, whereas validateAerospikeConfig is per rack. validateSCNamespaces has been created to avoid clutter in the main rack loop in the `validate()` function

Reply:
Both ways are ok, we have unconditional loop of racks to call validateAerospikeConfig() and similar loop we are using inside validateSCNamespaces(). This can be done either ways.